_posts/en/newsletters/2020-02-26-newsletter.md

```diff
@@ -0,0 +1,138 @@
+  quickly routing or rejecting an HTLCs and would ensure users of [hold
+  invoices][topic hold invoices] (for example,  Carol) paid the routing
+  nodes (such as Bob) for tying up their routing capital.
+
+    Several responders seemed to like the idea and began discussing how
+    it might be implemented as well as what technical challenges it
+    would need to overcome.
+
+- **LN direct messages:** Rusty Russell [proposed][russell dm] allowing
+  LN nodes to route encrypted messages between peers without using the
+  LN payments mechanism.  This could replace current uses of
+  messages-over-payments, such as [Whatsat][], with a simpler protocol
+  that might be easier to build upon (such as for the *offers* idea
+  described in [Newsletter #72][news72 offers]).  Russell's proposal
+  originally specified using the same there-and-back onion routing used
+  for LN payments (HTLCs), but developer ZmnSCPxj proposed having the
+  message sender specify the full path from their node to the message
+  recipient and back to the sender.  For example, if Alice wants to
+  communicate with Carol, she might choose the following path:
+
+        Alice → Bob → Carol → Dan → Alice
+
+    This type of circle routing would make surveillance more difficult
+    and would eliminate the overhead for routing nodes of having to
+    store the return path, making the protocol stateless for routing
+    nodes.  Discussion remains ongoing as of this writing, with a focus
+    on preventing the mechanism from being abused for spam and for
+    enhancing privacy.
+
+FIXME:jnwebery-sbc-report
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure.
+Please consider upgrading to new releases or helping to test release
+candidates.*
+
+- [Bitcoin Core 0.19.1][] (release candidate)
+
+- [LND 0.9.1][] (release candidate)
+
+FIXME:harding:update-links-tuesday-evening
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #13339][] wallet: Replace %w by wallet name in -walletnotify script FIXME:bitschmidty
+
+- [Eclair #1325][] allows the SendToRoute endpoint to accept routing
```

Comment:
Perhaps use code formatting for `SendToRoute`? (to be consistent with how we code format RPC methods for bitcoind)